addons/l10n_uy/models/l10n_latam_document_type.py

```diff
@@ -0,0 +1,30 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+import re
+
+
+class L10nAccountDocumentType(models.Model):
+
+    _inherit = 'l10n_latam.document.type'
+
+    internal_type = fields.Selection(selection_add=[('stock_picking', 'Delivery Guide')])
```

Comment:
We can add the selection option directly to l10n_latam_invoice_document module. do you think this it could be a problem? do you prefer that we leave it here?  @jco-odoo

Reply:
Ok. Done. I will create the PR for latam later